etl/steps/data/garden/regions/2023-01-01/migration.py

```diff
@@ -0,0 +1,884 @@
+        "Micronesia (country)",
+        "Nauru",
+        "New Caledonia",
+        "New Zealand",
+        "Niue",
+        "Norfolk Island",
+        "Northern Mariana Islands",
+        "Palau",
+        "Papua New Guinea",
+        "Pitcairn",
+        "Samoa",
+        "Solomon Islands",
+        "Tokelau",
+        "Tonga",
+        "Tuvalu",
+        "United States Minor Outlying Islands",
+        "Vanuatu",
+        "Wallis and Futuna",
+    ],
+    "South America": [
+        "Argentina",
+        "Bolivia",
+        "Brazil",
+        "Chile",
+        "Colombia",
+        "Ecuador",
+        "Falkland Islands",
+        "French Guiana",
+        "Guyana",
+        "Paraguay",
+        "Peru",
+        "South Georgia and the South Sandwich Islands",
+        "Suriname",
+        "Uruguay",
+        "Venezuela",
+    ],
+}
+
+# Define the list of continents.
+# These regions will be assigned the region type "continent".
+continents = sorted(continent_members)
+
+# Manually define other aggregate regions.
+aggregate_regions = {
+    "Channel Islands": [
+        "Guernsey",
+        "Jersey",
+    ],
+    "European Union (27)": [
+        "Austria",
+        "Belgium",
+        "Bulgaria",
+        "Croatia",
+        "Cyprus",
+        "Czechia",
+        "Denmark",
+        "Estonia",
+        "Finland",
+        "France",
+        "Germany",
+        "Greece",
+        "Hungary",
+        "Ireland",
+        "Italy",
+        "Latvia",
+        "Lithuania",
+        "Luxembourg",
+        "Malta",
+        "Netherlands",
+        "Poland",
+        "Portugal",
+        "Romania",
+        "Slovakia",
+        "Slovenia",
+        "Spain",
+        "Sweden",
+    ],
+    "Melanesia": [
+        "Fiji",
+        "Papua New Guinea",
+        "Solomon Islands",
+        "Vanuatu",
+    ],
+    "Polynesia": [
+        "American Samoa",
+        "Cook Islands",
+        "French Polynesia",
+        "New Zealand",
+        "Niue",
+        "Norfolk Island",
+        "Pitcairn",
+        "Samoa",
+        "Tokelau",
+        "Tonga",
+        "Tuvalu",
+        "Wallis and Futuna",
+    ],
+    "World": [
+        "Africa",
+        # 'Antarctica',
+        "Asia",
+        "Europe",
+        "North America",
+        "Oceania",
+        "South America",
+    ],
+}
+
+# Define the list of continents.
+# These regions will be assigned the region type "aggregate".
+aggregates = sorted(aggregate_regions)
+
+# Manually list regions whose official definition is unclear.
+# They will be assigned the region type "other".
+other_regions = [
+    "Abkhazia",
+    "Akrotiri and Dhekelia",
+    "Nagorno-Karabakh",
+    "Northern Cyprus",
+    "Serbia excluding Kosovo",
+    "Somaliland",
+    "South Ossetia",
+    "Svalbard and Jan Mayen",
+    "Transnistria",
+    "United States Minor Outlying Islands",
+]
+
+# Manually list regions to remove.
+regions_to_remove = [
+    # Is there any reason to keep these historical regions?
```

Comment:
Some datasets have data on these historical regions/states, before they became Germany and Italy — V-Dem is one prominent example. Would that be a reason to keep them?

Reply:
Probably not - the two main uses of this file for us are a) to compute aggregations in a more automated way and b) to join data across datasets (e.g. when doing a scatterplot). I doubt that we have enough datasets that contain comparable definitions for these entities that either case is compelling. We can always _use_ countries within one dataset as we like - removing them here just communicates that we don't view these as useful enough that for either aggregation or joining. Does that make sense?

Reply:
Thanks Daniel, that's also how I see it. In fact I have been very permissive letting regions stay in the dataset, maybe I should have removed other complicated regions. But for this specific case, if we are not expecting these historical regions to appear in any dataset apart from V-Dem, then there's no strong reason to keep them.
In any case, if we change our mind, we can add them in the future.

Reply:
Yes, that makes sense. I expect us to add more datasets with these historical states, and we may want to create some aggregates there — but the number of datasets probably will likely stay low.

Reply:
I don't think it's a problem to include those regions, but we'd simply need to add the necessary information (e.g. their end year). But we can do this as a separate issue, to avoid delaying this one. It would be better to add those regions once we have better tools in place (i.e. `harmonize` and `add_region_aggregates` are updated).